SUPPORT.md

```diff
@@ -34,7 +34,7 @@ Please see the [contributing guide](/CONTRIBUTING.md) for guidelines on how to b
 
 <!-- ## Commercial Support
 
-Commercial support for this project may be available. Please get in touch with [NGINX sales](https://www.nginx.com/contact-sales/) or check your contract details for more info! -->
+Commercial support for this project may be available. Please get in touch with [F5 (NGINX) sales](https://www.f5.com/products/get-f5/) sales or check your contract details for more info! -->
```

Comment:
Can you remove the parentheses in "F5 (NGINX)"?
```suggestion
Commercial support for this project may be available. Please get in touch with [F5 NGINX sales](https://www.f5.com/products/get-f5/) sales or check your contract details for more info! -->
```

Reply:
Yup, that's also consistent with our style guides, thank you.